orderer/consensus/etcdraft/consenter.go

```diff
@@ -232,19 +208,10 @@ func (c *Consenter) HandleChain(support consensus.ConsenterSupport, metadata *co
 		StreamsByType: cluster.NewStreamsByType(),
 	}
 
-	var haltCallback func() // called after the etcdraft.Chain halts when it detects eviction form the cluster.
-	if c.InactiveChainRegistry != nil {
-		// when we have a system channel, we use the InactiveChainRegistry to track membership upon eviction.
-		c.Logger.Info("With system channel: after eviction InactiveChainRegistry.TrackChain will be called")
-		haltCallback = func() {
-			c.InactiveChainRegistry.TrackChain(support.ChannelID(), nil, func() { c.ChainManager.CreateChain(support.ChannelID()) })
-			c.ChainManager.ReportConsensusRelationAndStatusMetrics(support.ChannelID(), types.ConsensusRelationConfigTracker, types.StatusInactive)
-		}
-	} else {
-		// when we do NOT have a system channel, we switch to a follower.Chain upon eviction.
-		c.Logger.Info("Without system channel: after eviction Registrar.SwitchToFollower will be called")
-		haltCallback = func() { c.ChainManager.SwitchChainToFollower(support.ChannelID()) }
-	}
+	// called after the etcdraft.Chain halts when it detects eviction form the cluster.
+	// when we do NOT have a system channel, we switch to a follower.Chain upon eviction.
+	c.Logger.Info("Without system channel: after eviction Registrar.SwitchToFollower will be called")
```

Comment:
why would we want to mention the system channel term in logs? and I'm also not sure how informative this message is. Can we instead explain what is going on?

Reply:
yes, makes sense, I changed it to

"After eviction from the cluster Registrar.SwitchToFollower will be called, and the orderer will become a follower of the channel."